_posts/2020-02-15-pmemkv-bindings.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Language bindings for pmemkv
+author: lukaszstolarczuk
+layout: post
+identifier: pmemkv-bindings
+---
+
+Along with [pmemkv][pmemkv_pmem_io] written in C/C++ there are a bunch of
+bindings for high-level languages. Currently: Java (with JNI), Node.js, Python and Ruby.
+
+Picture below properly summarize architecture and software stack of pmemkv and its bindings.
+![pmemkv_bindings_image](/assets/pmemkv_bindings.png)
+
+Always, the most current updates about bindings (and pmemkv in general) can be found on
+[pmemkv's README file][pmemkv_bindings_readme].
+
+### Common for bindings
+
+There are few common characteristics regarding all bindings:
+
+#### Work with libpmemkv 1.0
+
+All of our currently released bindings work properly with libpmemkv 1.0. Unfortunately their API
+may not be fully functionally equal to the native C/C++ API. These **bindings, which are
+in version 1.0** deliver API in accordance with **pmemkv 1.0**. In general, pmemkv's binding
+released in version X.Y is (always) functionally greater or equal to pmemkv in the same X.Y version.
+Additionally, if a binding's version is prior to 1.0 (any 0.x version), it should be considered
+an experimental implementation.
+
+Taking into consideration that libpmemkv is developing in the fast pace, there are always
+some gaps in the current versions of bindings' implementations. Even more, each binding
+is developed in its own pace, so the best way to check compatibility with pmemkv
+is to read changelog of the latest release of a specific binding.
+
+#### Handling pmemkv statuses
```

Comment:
This paragraph adds nothing to the post. In effect, what you are saying is that "there are some error states, they are handled differently for every language, see docs to check how".

I'd change this to something like "Idiomatic language support" in which you explain that we tried to make bindings feel natural to the language (idiomatic), and because of this we had to slightly divert from the original library in some areas, such as return values, error handling and so on.